src/Application/Regulation/Query/GetRegulationOrderHistoryQueryHandler.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Application\Regulation\Query;
+
+use App\Application\Regulation\View\RegulationOrderHistoryView;
+use App\Domain\Regulation\Repository\RegulationOrderHistoryRepositoryInterface;
+
+final class GetRegulationOrderHistoryQueryHandler
+{
+    public function __construct(
+        private RegulationOrderHistoryRepositoryInterface $regulationOrderHistoryRepository,
+    ) {
+    }
+
+    public function __invoke(GetRegulationOrderHistoryQuery $query): ?RegulationOrderHistoryView
+    {
+        $row = $this->regulationOrderHistoryRepository->findLastRegulationOrderHistoryByUuid($query->regulationOrderUuid);
+
+        if (!$row) {
+            return null;
+        }
+
+        $date = $row['date'];
+
+        return new RegulationOrderHistoryView(
+            date: $date->format('d-m-Y'),
```

Comment:
(Détail) Il vaut mieux faire le formatage des dates le plus tard possible, car ça peut dépendre du contexte du client (ex : des histoires de fuseau horaire, etc)

De plus ne veut-on pas afficher la date au format francophone, donc 01/02/2025 pour le 1er fevrier, plutôt que 01-02-2025 ?

Donc garder ici un DateTimeInterface

```suggestion
            date: $date,
```

Et faire le formatage dans Twig avec `date|date('d/m/Y')`